pkg/build/api/v1/types.go

```diff
@@ -303,6 +303,9 @@ type BuildStrategy struct {
 
 	// CustomStrategy holds the parameters to the Custom build strategy
 	CustomStrategy *CustomBuildStrategy `json:"customStrategy,omitempty" description:"holds parameters to the Custom build strategy"`
+
+	// ExternalStrategy holds the parameters to the External build strategy
+	ExternalStrategy *ExternalBuildStrategy `json:"externalStrategy,omitempty" description:"holds parameters to the External build strategy"`
```

Comment:
@bparees so openshift itself does not know how the final pipeline looks like? i mean if jenkinsfile defines it, how we are going to visualize it in the console? i was expecting something like a linked build configurations (prev step/next step), where some build configs will be external and some local (regular docker/s2i/custom builds)...